src/graph/validator/GoValidator.cpp

```diff
@@ -69,6 +69,23 @@ Status GoValidator::validateWhere(WhereClause* where) {
   }
 
   auto expr = where->filter();
+  auto filterExprs = ExpressionUtils::collectAll(
```

Comment:
Better to add a expression util function to avoid repeating the same code logic elsewhere, maybe another xxxValidator?

Reply:
fixed